tests/python/driver/tvmc/test_autoscheduler.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+import pytest
+import os
+import tarfile
+
+from os import path
+
+from tvm import auto_scheduler
+from tvm.driver import tvmc
+
+
+def _get_tasks(model):
+    mod, params = tvmc.frontends.load_model(model)
+    tasks, weights = tvmc.autoscheduler.get_tuning_tasks(mod, params, "llvm")
+    return (tasks, weights)
+
+
+def _autoscheduler_test_helper(
```

Comment:
I'm not sure if we need to actually run auto_scheduler in this test. It seems to me that this helper is used by the following 4 test cases. However, TVMC should not be in charge of testing "tasks", "tuning records", "early stopping" and "no tuning records", especially those tests don't even use TVMC commands.

Reply:
I removed this file. But now the autoscheduler is untested.